### ISSUE TITLE
Fix warning: Avoid using `history.pushState(...)` ...

### ISSUE THREAD

**Issue by cintek** [ISDuBA/ISDuBA#75]
When you open the application and go to `Home` for example, you can see the following warning in the browser console:
```Avoid using `history.pushState(...)` and `history.replaceState(...)` as these will conflict with SvelteKit's router. Use the `pushState` and `replaceState` imports from `$app/navigation` instead.```

**Comment by koplas** [comment]
`history.replaceState` gets called in the Keycloak dependency: https://github.com/keycloak/keycloak/blob/e765932df3221b249569a282f8b0b37bd3cca9b7/js/libs/keycloak-js/src/keycloak.js#L262

**Comment by ThomasJunk** [comment]
I don't know whether this poses a real problem (in any sense of the word "real").

1) We can't (easily) change how keycloak.js works
2) There are no visible problems for our application (detected)
3) AFAICS this should not affect us:
 - https://github.com/sveltejs/kit/issues/11671#issuecomment-1911754463 explains the rationale why this message appears at all
 - It seems to be that it is only a thing in dev-mode
 - We are not using the sveltekit routing system so there is IMHO no problem anyways.

tl;dr

I opt to `wontfix`.

**Comment by koplas** [comment]
Closing as won't fix, as the usage of `history.pushState` is inside of third-party code and doesn't affect our application.
